reconstruction/cvt/src/main/java/org/jlab/rec/cvt/banks/HitReader.java

```diff
@@ -337,6 +337,8 @@ public void fetch_SVTHits(DataEvent event, int omitLayer, int omitHemisphere, In
                 if (SvtStrip.getEdep() == 0) {
                     SvtStrip.setStatus(1);
                 }
+                //get status from ccdb
```

Comment:
This would override the strip status that is set at line 338. Is that the intention?

Related to this, should Edep=0 hits used or not?

Reply:
The strips with zero Edep should not have a passing status in the CCDB table.  This should be checked with Yuri. This was set for MC, where in the past this was implemented in GEMC to flag hits that should be removed.  I think that only hits with non-zero Edep should be used.

Reply:
It seems ADC=0 hits have Edep>0 for data. So overriding the check on Edep=0 is not having an effect